array_api_compat/common/_linalg.py

```diff
@@ -77,7 +77,8 @@ def matrix_rank(x: ndarray,
     # dimensional arrays.
     if x.ndim < 2:
         raise xp.linalg.LinAlgError("1-dimensional array given. Array must be at least two-dimensional")
-    S = xp.linalg.svd(x, compute_uv=False, **kwargs)
+    S = xp.linalg.svdvals(x, **kwargs)
+    #S = xp.linalg.svd(x, compute_uv=False, **kwargs)
```

Comment:
This change looks like it is causing the numpy array api tests to fail.

Reply:
The current numpy dev CI failures can be ignored (they should go away once https://github.com/numpy/numpy/pull/25668 is merged), but the other ones are important.

Reply:
Thanks for the catch. I think this should be equal to svdvals (if svdvals exists), so I just used a hasattr check. Lemme know if that's too hacky

Reply:
You should use `get_xp(xp).linalg.svdvals` so that it uses the svdvals alias. This pattern is already used in a few places in this file. 

As for not wrapping svdvals when it is present, we could do that too (I did this for `vector_norm` https://github.com/data-apis/array-api-compat/blob/916a84bafd2ae97c09d012616a13ee03354a1480/array_api_compat/numpy/linalg.py#L30-L33 but I forgot about svdvals).

Reply:
Thanks, updated.